src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Menu as MuiMenu, MenuProps as MuiMenuProps } from '@mui/material';
+
+import styled from '../../theme/styled';
+
+export interface MenuProps extends MuiMenuProps {
+  /**
+   * pass in menu items or buttons as children
+   * @default undefined
+   */
+  children: React.ReactNode;
+  /**
+   * an HTML element that's used to set the position of the menu
+   * @default undefined
+   */
+  anchorEl: HTMLElement | null;
+  /**
+   * if true, menu is shown
+   * @default undefined
+   */
+  open: boolean;
+}
+
+const MenuRoot = styled(MuiMenu)(() => ({
```

Comment:
nit: I think we want to start adding a hajimari label to these for easier debugging, so:
```
const MenuRoot = styled(MuiMenu, { label: 'Hajimari' })(() => ({
....
}));
```